caching/check.py

```diff
@@ -173,11 +174,14 @@ def create_repository_variable(name: str, linux_sha: str,
         sys.exit(1)
 
     VAR_NAME = get_workflow_name_to_var_name(args.workflow_name)
+    tree_name = args.workflow_name.split(' ', 1)[0]
 
     curr_sha = get_sha_from_git_ref(args.git_repo, args.git_ref)
     curr_clang_version = get_clang_version()
+    # pylint: disable-next=invalid-name
+    curr_patches_hash = get_patches_hash(tree_name)
     print(
-        f"Current sha: {curr_sha}\nCurrent Clang Version: {curr_clang_version}"
+        f"Current sha: {curr_sha}\nCurrent Clang Version: {curr_clang_version}\nCurrent patches hash: {curr_patches_hash}"
```

Comment:
For these types of prints, it might be cleaner to do something like this:

```python
    print(f"""\
Current SHA1: {curr_sha}
Current Clang Version: {curr_clang_version}
Current Patches Hash: {curr_patches_hash}
""")

Reply:
Sure thing, I added 734abf53d0b53d70473901cb7eb65bd3107544bc to accomplish this.